Ball.java

```diff
@@ -0,0 +1,18 @@
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball{" +
+                "color='" + color + '\'' +
+                ", number=" + number +
+                '}';
+    }
+

```

Comment:
remove the last empty line in all classes/interfaces/methods where it is present
```suggestion
```